tool/check-after-site-build.sh

```diff
@@ -10,7 +10,7 @@ if [[ -n $TRAVIS && $TASK != build* ]]; then
 fi
 
 travis_fold start check_links
-(set -x; ./tool/shared/check-links.sh)
+  (set -x; ./tool/shared/check-links.sh --external)
```

Comment:
I'm enabling external link check while I'm actively working on the site. Hopefully that is ok with you.

Reply:
Actually, it seems that we're hitting HTTP 429 warnings when we do this on every push. So I'll set it up to only run `--external` on cron jobs (which aren't setup yet, but will be soon).